chronosphere/notification_policy_data.go

```diff
@@ -66,6 +67,11 @@ type RoutesNotifierListNotifier struct {
 	Slug string `json:"slug,omitempty"`
 }
 
+// RoutesNotifierListGroupBy is the groupBy field
```

Comment:
nit: consider briefly describing what this field does, i.e. "RouteNotifierListGroupBy is a list of label names by which notifications are grouped".